src/unit-test/Ingredients.test.js

```diff
@@ -0,0 +1,18 @@
+import '../setupTest.js'
+import Ingredients from '../components/Ingredients';
+import fetch from '../unit-test/mocks/ingredientFetch';
+global.fetch = fetch;
```

Comment:
Why does it need to be global if you are using it in the same file where it is imported?

Reply:
global just makes sure the window uses the mock fetch and not the real one. Has to be set locally because the fetches are different. It's not global in the sense that it works across files if that's what you mean.

Reply:
ok